model/clusters_mgmt/v1/metric_queries_resource.model

```diff
@@ -27,4 +27,9 @@ resource MetricQueries {
 	locator SocketTotalByNodeRolesOS {
 		target SocketTotalByNodeRolesOSMetricQuery
 	}
+
+    //Reference to the resource that retrieves the cluster operator status metrics.
+	locator ClusterOperators {
+         target ClusterOperators
```

Comment:
Rename this to "ClusterOperatorsMetricQuery" as well: `target ClusterOperatorsMetricQuery`.